src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const Todoitem: React.FC<Props> = ({ todo }) => {
```

Comment:
```suggestion
export const TodoItem: React.FC<Props> = ({ todo }) => {
```
Much better for readabillity call React components var with UpperCamelCase
:))

Reply:
Thank you, I didn't see these two mistakes. I'll fix it